packages/quantic/docs/template/lwc-json/publish.js

```diff
@@ -104,9 +103,21 @@ function getMetadata(element) {
   return JSON.parse(xmlToJson(xmlData));
 }
 
+function getComponentCategory(element) {
+  return element.tags?.find((tag) => tag.originalTitle === 'category').value;
+}
+
+const categoryMap = {
+  search: 'Search',
+  resultTemplate: 'Result Template',
```

Comment:
I like the idea of having tags to categorize the components. But I'm not sure about the `Result Template` and `Utility` categories. I feel it would be clearer to developers if these tags could match the headless engine types (i.e., match the use cases). So `Search`, `Case Assist`, `Recommendation` are all different use cases, but everything related to result templates or utilities are shared between all use cases.

It looks like two different concepts to me. What do you think?

Reply:
Resolved in a slack conversation. Concensus is to make this change, allowing multiple categories to be specified. Given documentation is versioned, we could modify this later on if we find a case where this setup is no longer adequate as-is.